torch/_meta_registrations.py

```diff
@@ -2214,6 +2214,13 @@ def meta__adaptive_avg_pool2d_backward(grad_out, self):
     return self.new_empty(self.shape).to(memory_format=memory_format)
 
 
+@register_meta(aten._adaptive_avg_pool3d_backward)
+@out_wrapper()
+def meta__adaptive_avg_pool3d_backward(grad_output, self):
+    _adaptive_pool_empty_output_check(grad_output, "adaptive_avg_pool3d_backward")
+    return torch.empty_like(self, memory_format=torch.legacy_contiguous_format)
```

Comment:
See `adaptive_avg_pool3d_backward_cpu` (out shape) and `adaptive_avg_pool3d_backward_out_cpu_template` (shape check).